<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>图像处理助手</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/semiconductor.ico</normaloff>icons/semiconductor.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>391</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Micro USB设置</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>291</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="open_port">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="close_port">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>4</y>
        <width>291</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QComboBox" name="stopBit_comboBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>奇偶检验位</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="dataBit_comboBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="text">
          <string>串口</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="parity_comboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="baut_comboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>停止位</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="port_comboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>数据位</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="update_button">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/updates.jpg</normaloff>icons/updates.jpg</iconset>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>无线热点设置</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>271</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>远端IP</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="remotePort_editline"/>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>远端端口</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="localPort_editline"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="remoteIP_editline"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="localIP_editline"/>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>本地端口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>本地IP</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>271</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="listen_button">
         <property name="text">
          <string>开始接听</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="disconnect_button">
         <property name="text">
          <string>断开连接</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>9</y>
      <width>461</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>消息记录</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>16</y>
       <width>461</width>
       <height>401</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="overwriteMode">
      <bool>true</bool>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>461</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>串口输入窗口</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>14</y>
       <width>451</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="clearInput_button">
        <property name="text">
         <string>清除输入</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="send_button">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>换行</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QPlainTextEdit" name="plainTextEdit_2">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="overwriteMode">
         <bool>true</bool>
        </property>
        <property name="backgroundVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>数据接收</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="receive_button">
           <property name="text">
            <string>开始接收</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="stopRec_button">
           <property name="text">
            <string>停止接收</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>数据处理</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>231</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QPushButton" name="image_button">
           <property name="text">
            <string>图像生成</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>保存图像</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="file_button">
           <property name="text">
            <string>保存数据</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionconsult_help"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <action name="actionconsult_help">
   <property name="icon">
    <iconset>
     <normaloff>../Toolbar Icons ico/ico/About.ico</normaloff>../Toolbar Icons ico/ico/About.ico</iconset>
   </property>
   <property name="text">
    <string>查看帮助</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>localIP_editline</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_local_ip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localPort_editline</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_local_port()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remoteIP_editline</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_remote_ip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remotePort_editline</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>set_remote_port()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_newline()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionconsult_help</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>consult_help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_port</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_openPort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_port</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_closePort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>receive_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_rece()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopRec_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_stopRecv()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listen_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_listen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnect_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_disconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>port_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_port()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>baut_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_bautrate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBit_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_stopbit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataBit_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_databit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parity_comboBox</sender>
   <signal>activated(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>set_parity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearInput_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_clearInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>779</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>update_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>click_updatePort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_rece()</slot>
  <slot>click_stopBit()</slot>
  <slot>click_image()</slot>
  <slot>click_clearInput()</slot>
  <slot>click_send()</slot>
  <slot>click_openPort()</slot>
  <slot>click_closePort()</slot>
  <slot>click_disconnect()</slot>
  <slot>click_listen()</slot>
  <slot>set_local_ip()</slot>
  <slot>set_local_port()</slot>
  <slot>set_remote_ip()</slot>
  <slot>set_remote_port()</slot>
  <slot>set_newline()</slot>
  <slot>consult_help()</slot>
  <slot>set_port()</slot>
  <slot>set_bautrate()</slot>
  <slot>set_stopbit()</slot>
  <slot>set_databit()</slot>
  <slot>set_parity()</slot>
  <slot>click_stopRecv()</slot>
  <slot>click_updatePort()</slot>
 </slots>
</ui>
